### DISCUSSION TITLE
Stuck on Connecting to Wallet after reboot of Big Sur

### DISCUSSION THREAD

**Discussion by antwerpenR** [Chia-Network/chia-blockchain#1442] [Q&A]
My system has been running fine for a week or so with a dozen plots.  I rebooted the Mac (it did ask me did i want to stop farming and I said Yes - shutdown was normal) and after reboot Chia always hangs on Connecting to Wallet - why and what to do>

**Comment by hoffmang9** [comment]
Try deleting your wallet db and report back.

From Terminal:
`rm ~/.chia/mainnet/wallet/db/*`

**Reply by antwerpenR** [reply]
I did that.  On restarting it asked me to drag and drop Backup for color coins (I do not have or know what that is) - so I skipped to Safe.  Now it will not connect and plots show as unavailable,  What next?
<img width="1237" alt="Screenshot 2021-03-23 at 9 18 51 AM" src="https://user-images.githubusercontent.com/400794/112078371-44896e00-8bb9-11eb-83ed-57481e991c7f.png">

<img width="1237" alt="Screenshot 2021-03-23 at 9 18 59 AM" src="https://user-images.githubusercontent.com/400794/112078362-405d5080-8bb9-11eb-8eed-dcc7ca2d6ded.png">

**Reply by antwerpenR** [reply]
Can I safely delete the app and then reinstall, recreating the wallet from the Seed Phrase?  Will that allow all the plots to be recovered/reused?

**Reply by antwerpenR** [reply]
UPDATE.  I managed to recover by replacing the entire mainnet directory in ~/.chia with a backup from a few hours before the issue - deleting/replacing the wallet folder alone was NOT sufficient.  The app then started and showed it was Syncing....but it had to synchronise all the way up to Block 18600 or so.   Surely this is not normal behaviour?

Now it seems to be working again but I am concerned that if I quit and restart again, will I again have issues and/or need to resync all tbe blocks?  
![image](https://user-images.githubusercontent.com/400794/112105391-5124bb00-8be7-11eb-8625-fb6f5558e03d.png)

**Reply by jdelgadoesteban** [reply]
@hoffmang9 rm ~/.chia/mainnet/wallet/db/* fixed the issue for me many thanks.

My client shows full node status nodes and connections as spinning wheels as well though plotting seems to be working just fine.

**Reply by sean272727** [reply]
I had the exact same issue, rm ~/.chia/mainnet/wallet/db/* fixed the issue, now just spinning wheels. How do you replace the entire mainnet directory with a backup?

**Reply by abougu** [reply]
Exactly the same issue here - stuck on the spinning wheels. Any ideas?

**Reply by jdelgadoesteban** [reply]
I have been able to finish a plot however it shows as not available waiting for sync in yellow even when I see it has been correctly compressed and moved to the final location and all temp files deleted form the SSD.

Full Node Status, Blocks and Connections show a never ending spinning wheel.

**Reply by jdelgadoesteban** [reply]
> I had the exact same issue, rm ~/.chia/mainnet/wallet/db/* fixed the issue, now just spinning wheels. How do you replace the entire mainnet directory with a backup?

No idea unfortunately I had no back ups so I deleted the whole directory as suggested by another contributor, deleted the app, run the code above, restart, lunch the app, same issue, I can connect to the wallet, I see my plot but the rest is just spinning wheels.

**Reply by jdelgadoesteban** [reply]
UPDATE I have found the way of getting over the issue of the spinning wheels. On your Mac go to folder:
~/.chia and delete the contents of the folder "db" there are 2: blockchain_v1_mainnet.sqlite and peer_table_node.sqlite

After deleting those files and launching Chia again everything worked.

**Reply by abougu** [reply]
@jdelgadoesteban snap. I closed chia, deleted the two files in the /db folder and then re-opened the app. Chia had to sync the entire network again which took 10+ hours but now all working again.

**Reply by jdelgadoesteban** [reply]
Amazing, yeah it took me around 12 hours to sync back but now it´s all working, I am glad we managed to get there.

**Reply by dsyeag** [reply]
having the same issue on version 1.1.4. I tried uninstalling chia after getting hung on 'Connecting to wallet', but my plots were gone and chain starting back from 0 when I got back online. I tried restoring my /.chia file to get back to where I left off in terms of syncing, but that just brought me back to 'Connecting to wallet'. Is there any way to un-corrupt and restore my db file so that I don't have to resync/replot? @hoffmang9

**Comment by hoffmang9** [comment]
Please try this experimental installer - https://download-chia-net.s3-us-west-2.amazonaws.com/tests/Chia-1.0.4-rc1.dmg

**Reply by jdelgadoesteban** [reply]
Thanks @hoffmang9 I shall do that and report back, thanks for your help.

**Reply by hoffmang9** [reply]
So the alternate installer worked for you?

**Reply by jdelgadoesteban** [reply]
@hoffmang9 the 1.0.4 installer worked just fine for me thanks.

**Reply by TristanNewmanIos** [reply]
This didn't fix the "Connecting to wallet" loop issue for me

**Reply by jdelgadoesteban** [reply]
> This didn't fix the "Connecting to wallet" loop issue for me


I am sorry to hear that, I had the same problem as you this is what I did:

1. Deleted the app.
2. Deleted the whole directory in ~/Library/Application Support/Chia
3. Deleted the 2 mentioned files inside ~/.chia/db
4. Reinstalled the app.


https://download-chia-net.s3-us-west-2.amazonaws.com/tests/Chia-1.0.4-rc1.dmg

Good luck

**Reply by TristanNewmanIos** [reply]
Thank you! That worked!

**Reply by jdelgadoesteban** [reply]
Nice one!

**Reply by CoventureStudios** [reply]
I followed the instructions above (best of) and while it continues to spin "Connecting to Wallet", this is what appears under devtools:

DevTools failed to load SourceMap: Could not parse content for file:///Applications/Chia.app/Contents/Resources/app.asar/build/static/js/2.6bd1e003.chunk.js.map: Unexpected end of JSON input

Help... please

**Reply by jdelgadoesteban** [reply]
@CoventureStudios do you get that message once you follow the steps above and the reinstall the beta version of 1.04?

**Reply by CoventureStudios** [reply]
Yes. That's after I followed the steps and reinstalling

**Reply by jdelgadoesteban** [reply]
Oh no, I am so sorry I don´t really know what else to suggest.

**Comment by jdelgadoesteban** [comment]
I am in the middle of plotting so once I finish that plot I will download the installer and run it.

**Comment by thundervortex** [comment]
I am having this same issue with Macbook Pro 2014 OSx 10.14.6.  Had a major crash due to chia last night and system restarted.  When rebooting it is connecting to wallet.  I removed the DB as instructed above and same thing.  My plots status = Not available.  I have been "waiting for synchronization".  How do I know this is synchronizing and going to come back online?

**Reply by jdelgadoesteban** [reply]
My hardware is very similar to yours, my suggestion would be not to try to plot in parallel. It has been a pretty disappointing and experience for me with plots taking more than 24 hours on average on a relatively fast 1TB SSD.

Try deleting this: rm ~/.chia/mainnet/wallet/db/*

**Reply by thundervortex** [reply]
Thank you!  I attempted deleting mainet/wallet/db/* and this did not fix the issue.

I started on Sunday and had been able to make 10 Plots in paralel before my crash tues night, so my macbook pro has been rocking.  The error message in Chia, was something about and exception triggering another exception.  So I'm not quite giving up on parallel plotting.  I have yet to upgrade to the 1.0.4 release, but have that in my sites, as well as I'll be taking backup copies of the CHIA folder now to hopefully not end up waiting on a sync if this happens again.

I ended up deleting the entire ~/.chia folder and letting it recreate on the next startup.
It remembered my seed phrases, must be stored in a prefs file elsewhere, and I just had to spend the day doing a sync again.  This morning I am synce'd and back to farming!

**Reply by jdelgadoesteban** [reply]
Cool, nice one! I recommend you to have your phrase written manually on a piece of paper or electronically encrypted on a non-networked device.
